arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.h

```diff
@@ -425,7 +410,7 @@ class WeightedTwoSidedEnumerator {
     _right.setForbiddenEdges(std::move(forbidden));
   };
 
-  auto setEmitWeight(bool flag) -> void { _emitWeight = flag; }
+  auto setEmitWeight(bool) -> void {}
```

Comment:
The trouble is that we use so many template instantiations of different templates and so they all must have the same API. I will check if we still need it.

Reply:
The trouble is that the `YenEnumerator` is instantiated sometimes with the `TwoSidedEnumerator` and sometimes with the `WeightedShortestPathEnumerator`. The former is also used by unweighted k-shortest-paths and so needs the switch, since it must not emit weights in the latter case. But then the `WeightedShortestPathEnumerator` also needs the method, even if it does not do anything.
But I have now elimited the method from the `WeightedTwoSidedEnumerator`, which does not need it and which is no longer used in the `YenEnumerator`.